UI/data/locale/en-US.ini

```diff
@@ -845,3 +845,8 @@ OutputWarnings.MP4Recording="Warning: Recordings saved to MP4 will be unrecovera
 # deleting final scene
 FinalScene.Title="Delete Scene"
 FinalScene.Text="There needs to be at least one scene."
+
+# no sources
+NoSources.Title="No Sources"
+NoSources.Text.Stream="There are no sources! Are you sure you want to stream a blank scene?"
```

Comment:
I'd change it to something friendlier, and more helpful.

"It looks like you haven't added any sources to your scene, so you will only be recording a blank screen. Are you sure you want to do this?

You can add sources by clicking the + under the Sources box in the main window at any time."